### ISSUE TITLE
`null-ls-builtins` help entry

### ISSUE THREAD

**Issue by zdufour-asp** [jose-elias-alvarez/null-ls.nvim#1057]
### Issues

- [X] I have checked [existing issues](https://github.com/jose-elias-alvarez/null-ls.nvim/issues) and there are no existing ones with the same request.

### Feature description

IDK anything about how the `help` system works in vim / neovim (I've only been off vscode for 8 months), but it'd be nice to have a `null-ls-builtins` help entry so that I don't need to leave my editor

### Help

Yes, but I don't know how to start. I would need guidance

### Implementation help

The main thing I'd want input from a maintainer on is what to do about [BUILTINS.md](https://github.com/jose-elias-alvarez/null-ls.nvim/blob/main/doc/BUILTINS.md) if we add a vim help formatted version of the same document.

There's a chance that I can use something like [md2vim](https://github.com/FooSoft/md2vim) to convert `BUILTINS.md` to a vimdoc help file (and could set up a github action to autobuild that), but if that isn't the case, should `BUILTINS.md` be:

1. changed to a redirect
2. left as-is with a redirect message added
3. maintained separately (seems like a lot of work)
4. Left as-is, and we scrap this idea, because it's not really that big of a deal to open a browser

**Comment by jose-elias-alvarez** [comment]
I agree that maintaining this separately would be a ton of work. Check out #949 for an example of an automated approach. If you're interested in picking it up, we could start by converting `BUILTINS.md`. It's already automatically generated, so getting it to look nice shouldn't take too much work.
